WORKSPACE

```diff
@@ -13,7 +13,7 @@ rules_js_dependencies()
 
 load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies", "register_coreutils_toolchains", "register_jq_toolchains")
 
-aspect_bazel_lib_dependencies(override_local_config_platform = True)
+aspect_bazel_lib_dependencies()
```

Comment:
Do you need to also set up any toolchains?

Reply:
Maybe? It looks like we'll also have to drop testing with Bazel 5